backend/app/assets/stylesheets/spree/backend/components/_number_with_currency.scss

```diff
@@ -0,0 +1,34 @@
+.number-with-currency {
+  &-symbol {
+    min-width: 2.5em;
+    text-align: center;
+  }
+
+  &-amount {
+    text-align: right;
+  }
+
+  &-addon {
+    width: 5.5rem;
+    text-align: left;
+  }
+
+  & &-select {
+    padding: 0;
+    background: #fff;
+
+    select {
+      @extend .c-select;
+
+      display: inline-block;
+      width: 100%;
+      height: 31px;
+      border: 0;
+
+      &::-ms-expand {
+        // Required to see full text of selected value on IE11
+        display: none;
```

Comment:
Bootstrap [already takes care of hiding this](https://github.com/solidusio/solidus/blob/master/backend/vendor/assets/stylesheets/solidus_admin/bootstrap/_custom-forms.scss#L154).